searchbox.py

```diff
@@ -0,0 +1,169 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import folium
+from folium.plugins import FastMarkerCluster
+from streamlit_folium import st_folium
+from ECHO_modules.get_data import get_echo_data # Import the get_echo_data function, which is the function that does the work of retrieving data from the SBU database
+from ECHO_modules.get_data import get_spatial_data # Import this function, which will help us get county boundaries
+from ECHO_modules.geographies import spatial_tables, fips, region_field, states # Import for mapping purposes
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+
+# Styles for States ("other") and selected regions (e.g. Zip Codes) - "this"
+map_style = {'this': {'fillColor': '#0099ff', 'color': '#182799', "weight": 1},
+'other': {'fillColor': '#FFA500', 'color': '#182799', "weight": 1}}
+
+# Get list of counties in the format COUNTY, STATE
+url = "https://raw.githubusercontent.com/edgi-govdata-archiving/"
+url += "ECHO_modules/packaging/data/state_counties_corrected.csv"
+all_counties = pd.read_csv( url )
+column_1 = 'County'
+column_2 = 'FAC_STATE'
+all_counties['County_Pairs'] = all_counties[column_1] + ', ' + all_counties[column_2]
+county_list = all_counties['County_Pairs']
+county_list = county_list.unique()
+
+user_selection = st.selectbox(
+    'Search for a county',
+    county_list)
+comma_pos = user_selection.find(',')
+selected_state = user_selection[comma_pos + 2:]
+selected_county = user_selection[:comma_pos]
+
+
+def format_county(selected_county):
+    '''
+    Returns the county with the first letter of each word capitalized.
+
+    Example: format_county('SAN FRANCISCO') returns 'San Francisco'
+    '''
+    res = ''
+    county_words = selected_county.split(' ')
+    for word in county_words:
+        res += word.lower().capitalize() + ' '
+    return res[:-1]
+
+county_name = format_county(selected_county)
+
+
+st.header('Get Facilities in Selected County')
+#sql = 'select * from "ECHO_EXPORTER" where "FAC_COUNTY" = \'TOMPKINS\' and "FAC_STATE" = \'NY\'' # "ECHO_EXPORTER" contains basic info about all regulated industrial facilities, including location
```

Comment:
can delete this since it's hardcoded to tompkins

Reply:
will do!